cloudimg/aws.py

```diff
@@ -524,6 +566,9 @@ def register_image(self, snapshot, metadata):
             SriovNetSupport=metadata.sriov_net_support,
             BillingProducts=metadata.billing_products
         )
+        if metadata.tags:
+            self.tag_image(image, metadata.tags)
```

Comment:
We ignore return value from the method (`return image.create_tags(**attrs)`)

And it is not clear to be from AWS documentation what is returned (exception raised?) in case of error
* https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services
/ec2.html?highlight=create_tags#EC2.Image.create_tags
* https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateTags.html

I can just see there just a comment for `DryRun`
```
If you have the required permissions, the error response is DryRunOperation . Otherwise, it is UnauthorizedOperation .
```

If exception is raised than we should be OK; but if it returns empty list than we have a problem
```
Returns

    A list of Tag resources
```
because one would expect image is successfully  "annotated" with tags but we could continue
even without that. And next time we would not be able to find image by tags.

Reply:
From what I understood it will raise, as in `DryRun` it says "Otherwise, it is UnauthorizedOperation", it is, without the `DryRun` option it will raise the `UnauthorizedOperation` exception in case of error.

Reply:
OK;

Could we at least log result to debug logs? Just in case :-)

Reply:
Sure, I've added it, PTAL again